enhancements/network/network-policy-audit-logging.md

```diff
@@ -0,0 +1,315 @@
+---
+title: audit-logging-of-network-policy-events
+authors:
+  - "@astoycos"
+reviewers:
+  - "@abhat" 
+  - "@vpickard"
+  - "@trozet" 
+  - "@Billy99"
+approvers:
+  - "@knobunc"
+creation-date: 2020-12-11
+last-updated: 2020-12-11
+status: implementable
+
+---
+
+# Audit Loggging of Network Policy Events 
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The OVN-Kubernetes CNI uses [OVN](https://www.ovn.org) to implement node overlay networks for Kubenetes. When Ovn-Kubernetes is used as the CNI for an openshift cluster OVN ACLs are the used to implement Kubernetes NetworkPolicies.  ACL's can either allow or deny traffic by matching on packets with specific rules.  Built into the OVN ACL feature is the ability to specify logging for each "allow" or "deny" rule.  The enhancement will activate this logging and allow the customer to manipulate the logging level, rate, and namespaces in which it is used, therefore showing valiuable realtime information involving NetworkPolicies 
+
+## Motivation
+
+Currently openshift has no ability to audit security events reguarding NetworkPolicies, and there are many customers which require this feature for security policy and regulation. The logging will allow customers to monitor NetworkPolicy events and identify pattterns involving malicious activity with both allow and deny events. This is necessary in scenarios where customers require a certain level of compliance, such as monitoring firewall activity, intrusion detection support, or fo post-mortem analysis 
+
+### Goals
+
+- Activate configurable ACL allow/deny logging in Ovn-Kubernetes on a per namespace basis 
+- Allow the Customer to control the logging rate for this feature via the Cluster Network Operator's config 
+- Allow the Customer to configure this fetaure's use and level via a Namespace's `.metadata.annotations` field 
+- Collect the relevant data from the ovn-controller logs and preset it to the Customer in an seamless manner 
```

Comment:
```suggestion
- Collect the relevant data from the `ovn-controller` logs and present it to the cluster administrator 
```

I like using "cluster administrator" here as it more clearly identifies the target persona we're looking to support with this feature.